packages/batch-submitter/src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -587,14 +583,11 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
             oldBlockNumber: ele.blockNumber,
             newBlockNumber: latestBlockNumber,
           })
-          fixedBatch.push({
```

Comment:
Is it possible for this to be left as `MAX_SAFE_INTEGER` and then cause all subsequent transactions to be non-monotonic?

Reply:
It is not possible to submit blocks from the future in the CTC - https://github.com/ethereum-optimism/optimism/blob/baa3b761cd5693dd57b1acf7b4bd42500a83b34f/packages/contracts/contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol#L1072-L1074

Reply:
It seems like an additional possible way to have errors when attempting to send the transaction, the gas estimation will fail

Reply:
Yes definitely agree. We need to add tests to the auto healers I think at this point because they are being used frequently now. 

Are there changes that you’d like me to make before merging this and getting the BS up and running again @tynes ? Maybe add an issue for adding tests to the auto heal options?

Reply:
I think using a safe value for the blocknumber would be ideal that wouldn't cause the transaction to revert